PilotDataPlatform/metadata
# PILOT-1567 Add favourites authorization logic

## Summary

- Add logic to ensure that the create favourites endpoint does not present any vulnerabilities or expose any unauthorized information to users.
- Half of the logic is done in BFF: https://github.com/PilotDataPlatform/bff-web/pull/119

## JIRA Issues

https://indocconsortium.atlassian.net/browse/PILOT-1567

## Type of Change


## Testing

Are there any new or updated tests to validate the changes?


## Test Directions

Visual code review. Check logic against [instructions](https://indocconsortium.atlassian.net/browse/PILOT-1405?focusedCommentId=42862).
